src/gui/rss/automatedrssdownloader.h

```diff
@@ -105,6 +116,7 @@ private slots:
     QRegExp *m_episodeRegex;
     QShortcut *editHotkey;
     QShortcut *deleteHotkey;
+    QSet<QPair<QString, QString >> m_treeListEntries;
```

Comment:
`uncrustify.cfg` is adding the extra space. Might be an idea to change that and then I can re-run on these.